ConwaysGameOfLife/OutputFormatter.cs

```diff
@@ -10,11 +10,12 @@ public static class OutputFormatter
         public static string FormatUniverse(Universe universe) // universe created with user input - dimensions & live cells ?? - needed??
         {
             StringBuilder stringBuilder = new StringBuilder();
-            for (var y = 0; y < universe.GridLength; y++)
+            for (var y = 0; y < universe.GridLength; y++)// TODO: problem here?? - should width be first loop?

```

Comment:
we have tests that validate that this function is working as intended. it's easy to get the x,y mixed up but we can fall back on our existing tests to realise that this function is working.